locations/linked_data_parser.py

```diff
@@ -42,8 +42,12 @@ def find_linked_data(response, wanted_type) -> {}:
     def parse_ld(ld) -> GeojsonPointItem:
         item = GeojsonPointItem()
 
-        if geo := ld.get("geo"):
-            if isinstance(ld["geo"], list):
+        if (
```

Comment:
I'm not sure this is part of the spec, or a common problem. If not I'd prefer it moved into spider code. We'd have to stop using `LinkedDataParser.parse` instead use `LinkedDataParser.find_linked_data`, fix the result, then use `LinkedDataParser.parse_ld`, [eg](https://github.com/alltheplaces/alltheplaces/blob/ebca0409ede42f60c1c1f4a91a029817edeb9c89/locations/spiders/sunglass_hut.py#L18)

If it is common and we stick with this we need to fix `if isinstance(ld.get("geo"), list):` to `if isinstance(geo, list):`

Reply:
@Cj-Malone I ran it through the [schema.org validator](https://validator.schema.org/#url=https%3A%2F%2Flocations.modpizza.com%2Fusa%2Fky%2Flexington%2F2217-war-admiral-way) and it reported no errors, so I am guessing it is valid?  

Happy to fix it up either way!

Reply:
OK, yeah you're right, it's to spec. [location](https://schema.org/location) can be a [Place](https://schema.org/Place) and that can contain geo.

So we just need to do:

```patch
-if isinstance(ld.get("geo"), list):
+if isinstance(geo), list):
```